src/main/scala/com/gu/support/workers/exceptions/RetryImplicits.scala

```diff
@@ -44,6 +45,10 @@ object RetryImplicits {
     }
   }
 
+  implicit class ZuoraCatalogueConversions(val throwable: CatalogDataNotFoundException) {
```

Comment:
nit: could we have consistent spelling between catalog/catalogue?

Reply:
Yep. 👍 I corrected back to catalog a few times but missed this one